plugins/modules/docker_container_exec.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/python
+#
+# Copyright 2016 Red Hat | Ansible
```

Comment:
is it the standard header to set ?

Reply:
LOL, no, it's something I forgot to replace :D

Reply:
```suggestion
# Copyright (c) 2021, Felix Fontein <felix@fontein.de>
```

Reply:
I've fixed it (and adjusted the dates in two other copyright lines).